lib/assessment.rb

```diff
@@ -13,14 +14,18 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce {0, :+}
+    # I was looking for a solution based on info I found here:
+    # http://stackoverflow.com/questions/1538789/how-to-sum-array-of-numbers-in-ruby
+    # and the enumerable exercise we did with fibonacci in class yesterday
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.select(&:odd?)
+    # I based this answer off of the enumerables exercise we did in class yesterday with fibonacci numbers to find the sum of odd numbers: fibs.select(&:odd?).reduce(&:+), "\n"
```

Comment:
Good thinking, and your answer is close, but `select` is not the right tool. Can you think of one that might be a better fit?